Unknown privilege "ALL PRIVILEGES"

This project seems like a great idea, but I've hit a showstopper with the first dump command.

```console
panic: BUG: Unknown privilege "ALL PRIVILEGES"

goroutine 1 [running]:
github.com/SnoozeThis-org/pgperms.(*privilegeSet).Add(0x14000150bb8, {0x100cd2a1e, 0xe})
        github.com/SnoozeThis-org/pgperms/privileges.go:382 +0x12c
github.com/SnoozeThis-org/pgperms.mergePrivileges({0x140001f4000, 0x2a, 0x1400007f6b0?})
        github.com/SnoozeThis-org/pgperms/privmerge.go:27 +0xc54
github.com/SnoozeThis-org/pgperms.Dump({0x100e88750?, 0x14000024090?}, 0x14000100136?)
        github.com/SnoozeThis-org/pgperms/dump.go:19 +0x58
main.main()
        github.com/SnoozeThis-org/pgperms/cmd/pgperms/cli.go:58 +0x638
```

I see `ALL PRIVILEGES` isn't included in the structures of valid privileges. Do you plan to expand to this, in which case I may make a PR, or is the exclusion of `ALL PRIVILEGES` a design decision?

I'm only using `ALL PRIVILEGES` in my development database, but I'm not going to test this out in a production or staging environment, so this is blocking my adoption of the tool.

--- a/databases/postgres/pgperms.nix
+++ b/databases/postgres/pgperms.nix
@@ -5,19 +5,19 @@
 
 let
   pname = "pgperms";
-  version = "0.1.0";
+  version = "0.1.1";
   src = fetchFromGitHub {
     name = "${pname}-${version}-src";
     owner = "SnoozeThis-org";
     repo = pname;
-    rev = "9e23a87bece464ed54e5206a24fbba0270cf5c2f";
-    hash = "sha256-obUXIL59BiKNP45h2nU9wwWTsFgIDlPDLFlmHlzqeFE=";
+    rev = "a50358c5ba504ad2d6744fb63c6db04959814c96";
+    hash = "sha256-S1+cBmHSwHIOv+FLSfpGflXyAKTDnsgmth7YhugFpCI=";
   };
 in
 buildGoModule {
   inherit pname version src;
 
-  vendorHash = "sha256-X1IB1vRK1yJfzEkl40ZH7kw2r61WfMYNaEhyDpifPSQ=";
+  vendorHash = "sha256-LV1Lndbh1UGaukgBVAiHdcqZVyE19LpC+asErVORbfU=";
 
   doCheck = false;
 

